Makefile

```diff
@@ -52,7 +52,7 @@ tfgen::
 	go install -ldflags "-X github.com/pulumi/pulumi-gitlab/pkg/version.Version=${VERSION}" ${PROJECT}/cmd/${TFGEN}
 
 lint::
-	golangci-lint run
+	#golangci-lint run
```

Comment:
this matches our other repos.  We don't lint these as our generators do not produce clean lintable code.